cx-portal-shared-components/src/components/basic/Dropzone/old_deprecated/index.tsx

```diff
@@ -176,12 +176,14 @@ class DropzoneOldDEPRECATED extends React.Component<
     fileWithMeta.meta.percent = 0
     this.handleChangeStatus(fileWithMeta)
     this.forceUpdate()
-    this.uploadFile(fileWithMeta)
+    void this.uploadFile(fileWithMeta)
   }
 
   // expects an array of File objects
   handleFiles = (files: File[]) => {
-    files.forEach((f, i) => this.handleFile(f, `${new Date().getTime()}-${i}`))
+    files.forEach(
+      (f, i) => void this.handleFile(f, `${new Date().getTime()}-${i}`)
```

Comment:
use `Date.now()` instead of `new Date().getTime()`

Reply:
@oyo Made this change